src/player.js

```diff
@@ -98,8 +102,15 @@
       return this._playbackRate;
     },
     set playbackRate(value) {
+      if (value == this._playbackRate) {
+        return;
+      }
       var oldCurrentTime = this.currentTime;
       this._playbackRate = value;
+      this._startTime = null;
+      if (!this.paused) {
```

Comment:
> I don't think we should do this when we are idle? what is this.paused?

Player.paused is an old concept. It should be replaced with playState == 'paused' but playState currently depends on it. I can remove it in a follow-up patch. I need to do a sweep of our whole API and make sure it's consistent with the spec.

I think you are correct: we shouldn't do this if we are idle.